### PR TITLE
Add nonsense

### LINKED ISSUE #2: Test ToDo Block
What to do, test for Taylor

### PR DESCRIPTION
**Pull request by Virtruvian1**

### CODE REVIEW

File: CSV Parse/CSV Parse.csproj [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
+///commetnt fweajnauhfoqewrgou;nera
```
Comment (Virtruvian1) [inline CSV Parse/CSV Parse.csproj:2]: Why did you change this?

Comment (Virtruvian1) [reply CSV Parse/CSV Parse.csproj:2]: I did it because i can

**Review by taylor-d-jackson — CHANGES_REQUESTED** [review]
Nonsense Nonsense

**Comment by Virtruvian1** [comment]
done did your nonsense change

**Review by taylor-d-jackson — APPROVED** [review]
Approved yo!
